skimage/_shared/version_requirements.py

```diff
@@ -4,6 +4,30 @@
 import sys
 
 
+def check_python_version(min_version):
+    if not isinstance(min_version, tuple):
```

Comment:
I'm impartial to this, but it helps catch the case where you don't specify the python version as a tuple, but just as an int